### DISCUSSION TITLE
folder structure error

### DISCUSSION THREAD

**Discussion by zzzgit** [rollup/plugins#1502] [Q&A]
```
ts 5
nodejs 12
```

In tsconfig.json:

<img width="240" alt="image" src="https://user-images.githubusercontent.com/1060733/236140066-bc09633d-3937-43e4-bf47-755cbac568ec.png">

The resulting structure is like this: 

<img width="207" alt="image" src="https://user-images.githubusercontent.com/1060733/236142118-af52c84c-41e2-48e4-bcf6-fc71a3627da2.png">

It's wired. Why is `t` under `lib`?

**Comment by tada5hi** [comment]
I'm not sure if this issue is related to rollup. Can you provide more details or a repo for reproduction ?
